vtex-sites/storecomponents.store
# Feat: Adds tests for Homepage Flows

## What's the purpose of this pull request?
Adds e2e tests for common home page flows. Tests links and login

## How to test it?
Using either a production or a dev build just run `yarn cy:open` and run the desired files or `yarn cy:run` to run all tests on the CLI. In either case the `baseUrl` on `cypress.json` should be the path to where the store is running or the option `--config baseUrl` should be passed to the CLI.
